src/test/ts/browser/LoadTinyTest.ts

```diff
@@ -1,129 +1,120 @@
-import { Chain, Log, Pipeline, Assertions } from '@ephox/agar';
-import { UnitTest } from '@ephox/bedrock-client';
-import { Arr, Strings, Global } from '@ephox/katamari';
-import { SelectorFilter, Attribute, SugarElement, Remove } from '@ephox/sugar';
+import { Assertions } from '@ephox/agar';
+import { context, describe, it } from '@ephox/bedrock-client';
+import { Global } from '@ephox/katamari';
+import { pRender, remove } from '../alien/Loader';
+import { cleanupGlobalTinymce, VALID_API_KEY } from '../alien/TestHelper';
 
-import { cRender, cRemove } from '../alien/Loader';
-import { ScriptLoader } from '../../../main/ts/ScriptLoader';
+describe('LoadTinyTest', () => {
 
-UnitTest.asynctest('LoadTinyTest', (success, failure) => {
-  const cDeleteTinymce = Chain.op(() => {
-    ScriptLoader.reinitialize();
-
-    delete Global.tinymce;
-    delete Global.tinyMCE;
-
-    const hasTinymceUri = (attrName: string) => (elm: SugarElement<Element>) =>
-      Attribute.getOpt(elm, attrName).exists((src) => Strings.contains(src, 'tinymce'));
-
-    const elements = Arr.flatten([
-      Arr.filter(SelectorFilter.all('script'), hasTinymceUri('src')),
-      Arr.filter(SelectorFilter.all('link'), hasTinymceUri('href')),
-    ]);
-
-    Arr.each(elements, Remove.remove);
-  });
-
-  const cAssertTinymceVersion = (version: '4' | '5' | '6' | '7') => Chain.op(() => {
+  const AssertTinymceVersion = (version: '4' | '5' | '6' | '7') => {
     Assertions.assertEq(`Loaded version of TinyMCE should be ${version}`, version, Global.tinymce.majorVersion);
-  });
-
-  Pipeline.async({}, [
-    Log.chainsAsStep('Should be able to load local version of TinyMCE using the tinymceScriptSrc prop', '', [
-      cDeleteTinymce,
+  };
 
-      cRender({}, `
+  context('LoadTinyTest', () => {
+    it('Should be able to load local version of TinyMCE 7 using the tinymceScriptSrc prop', async () => {
+      await pRender({}, `
         <editor
           :init="init"
           tinymce-script-src="/project/node_modules/tinymce-7/tinymce.min.js"
         ></editor>
-      `),
-      cAssertTinymceVersion('7'),
-      cRemove,
-      cDeleteTinymce,
+      `);
 
-      cRender({}, `
+      AssertTinymceVersion('7');
+      remove();
+      cleanupGlobalTinymce();
```

Comment:
Put these common actions into `beforeEach` block

Reply:
Updated in this [commit](https://github.com/tinymce/tinymce-vue/pull/434/commits/6bb5d8f1f41a7d0458ed7889335c20a06de33c71) for review.